packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -527,6 +527,18 @@ contract SphinxUtils is SphinxConstants, StdUtils {
      *         proposals, broadcasting, etc).
      */
     function validate(SphinxConfig memory _config) external pure {
+        if (
+            _config.owners.length == 0 &&
+            _config.threshold == 0 &&
+            bytes(_config.projectName).length == 0 &&
+            _config.mainnets.length == 0 &&
+            _config.testnets.length == 0
+        ) {
+            revert(
+                "Sphinx: Detected an empty 'sphinxConfig' struct. Did you forget to add fields to it in your script's setUp function or constructor? If you've already added fields to it in your setUp function, have you called 'super.setUp()' in any contracts that inherit from your script?"
+            );
```

Comment:
I think we may want to use `\n` to avoid words getting wrapped weirdly in the user's terminal